### PR TITLE
Adding Game of Thrones Board Game

### PR DESCRIPTION
**Pull request by rodrigoimenes**
Submitting this PR means you have:

- [x] searched the [readme](../readme.md) file and searched the [open pull requests](https://github.com/edm00se/awesome-board-games/pulls) (no duplicates please)
- [x] and will follow [the desired format](../formatting.md)

Do you have further questions? Check the [contribution guidelines](../contributing.md) or [open an issue](https://github.com/edm00se/awesome-board-games/issues/new).

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -88,6 +88,15 @@
 | ------: | -------: | -------: |
 |   1 - 5 |       14 | 120-180m |
 
+### [Game of Thrones](https://en.wikipedia.org/wiki/A_Game_of_Thrones_(board_game)
+
+> A Game of Thrones allows the players to take on the roles of several of the Great Houses vying for control of the Seven Kingdoms, including House Stark, House Lannister, House Baratheon, House Greyjoy, House Tyrell, and as of the expansion A Clash of Kings, House Martell. Players maneuver armies to secure support in the various regions that comprise the Seven Kingdoms, with the goal of capturing enough support to claim the Iron Throne. The basic gameplay mechanics are reminiscent of Diplomacy, especially in the order-giving process, though A Game of Thrones is significantly more complicated overall.
+
+![Game of thrones boardgame image](https://awoiaf.westeros.org/index.php/File:Game-board-agot.png)
```
Comment (edm00se) [inline readme.md]: This link appears to be to the file resource page for the wiki. Changing the image to the closest size appropriate would be best.

```suggestion
![Game of thrones boardgame image](https://awoiaf.westeros.org/thumb.php?f=Game-board-agot.png&width=250)
```

**Comment by edm00se** [comment]
Thanks for contributing.

**Comment by rodrigoimenes** [comment]
How can i get a Hacktoberfest tag?

**Comment by edm00se** [comment]
Well, as far as this PR goes (internal to this repo), we can tag the issue so labelled with #10. As for the tracking for Hacktoberfest, provided you have signed up at [the official Hacktoberfest page](https://hacktoberfest.digitalocean.com/), it will track this public Pull Request and count it towards your total. You can log in with your GitHub account there and view your progress as well.

https://hacktoberfest.digitalocean.com/

**Pull request merged**